snapcraft/internal/errors.py

```diff
@@ -53,35 +53,44 @@ def __init__(self, step):
 class StepOutdatedError(SnapcraftError):
 
     fmt = (
-        'Failed to reuse files from previous build: '
+        'Failed to reuse files from previous run: '
         'The {step!r} step of {part!r} is out of date:\n'
         '{report}'
-        'To continue, clean that part\'s {step!r} step, run '
+        'To continue, clean that part\'s {step!r} step by running '
         '`snapcraft clean {parts_names} -s {step}`.'
     )
 
-    def __init__(self, *, step, part,
-                 dirty_properties=None, dirty_project_options=None,
+    def __init__(self, *, step, part, dirty_properties=None,
+                 dirty_project_options=None, changed_dependencies=None,
                  dependents=None):
         messages = []
         if dirty_properties:
-            humanized_properties = formatting_utils.humanize_list(
-                dirty_properties, 'and')
-            pluralized_connection = formatting_utils.pluralize(
-                dirty_properties, 'property appears',
-                'properties appear')
-            messages.append(
-                'The {} part {} to have changed.\n'.format(
-                    humanized_properties, pluralized_connection))
+                humanized_properties = formatting_utils.humanize_list(
```

Comment:
why the over-indent here?

Reply:
I didn't tell you I decided we should start using 8 spaces as our indentation?